test/test-conn.py

```diff
@@ -210,7 +210,15 @@ def handle_auth():
         logging.info('Data_item %d will send every %d seconds', data['sensor_id'], interval)
```

Comment:
in the commit f41f7855d0e8e167d4d80bb88d01a08c05fb54c8:

Resend schema if receive an error in script test-conn

Reply:
done